### PR TITLE
downgrade spring 6.1.4

### PR DESCRIPTION
**Pull request by maggarwal13**

### CODE REVIEW

File: whois-commons/src/main/resources/applicationContext-commons.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <context:component-scan base-package="net.ripe.db.whois.common"/>
 
     <context:annotation-config/>
-    <cache:annotation-driven mode="aspectj"/>
```
Comment (eshryane) [inline whois-commons/src/main/resources/applicationContext-commons.xml:24]: Do we not still need this?

File: whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/SchedulerConfig.java [hunk 2]
```diff
@@ -17,9 +18,9 @@
 import javax.sql.DataSource;
 
 @Configuration
-// @EnableAspectJAutoProxy(proxyTargetClass = true)
+@EnableAspectJAutoProxy(proxyTargetClass = true)
 @EnableScheduling
-@EnableSchedulerLock(defaultLockAtMostFor="23h", mode = AdviceMode.ASPECTJ)
```
Comment (eshryane) [inline whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/SchedulerConfig.java:22]: Also is it better to be explicit?

File: whois-commons/src/main/resources/applicationContext-commons.xml (hunk 1 above)
Comment (maggarwal13) [reply whois-commons/src/main/resources/applicationContext-commons.xml:24]: I wanted it to be as it was before spring upgrade.

File: whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/SchedulerConfig.java (hunk 2 above)
Comment (maggarwal13) [reply whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/SchedulerConfig.java:22]: I wanted it to be as it was before spring upgrade.

File: whois-commons/src/main/resources/applicationContext-commons.xml (hunk 1 above)
Comment (eshryane) [reply whois-commons/src/main/resources/applicationContext-commons.xml:24]: I think it's safer to be explicit, we weren't doing that before, which means we may have mixed AspectJ weaving and proxying. We should specify aspectj mode everywhere.

Comment (eshryane) [reply whois-commons/src/main/resources/applicationContext-commons.xml:24]: Have a read of the issue https://github.com/spring-projects/spring-framework/issues/32882

"I still recommend revising your application setup towards exclusive weaving usage (no proxy setup at all)."

I think we're in the same situation.

Comment (eshryane) [reply whois-commons/src/main/resources/applicationContext-commons.xml:24]: Agreed lets downgrade completely and add mode later when we re-upgrade

**Review by eshryane — APPROVED** [review]

File: whois-commons/src/main/resources/applicationContext-commons.xml (hunk 1 above)
Comment (maggarwal13) [reply whois-commons/src/main/resources/applicationContext-commons.xml:24]: Yes,  lets do it as part of spring upgrade.Thanks

**Pull request merged**